### ISSUE TITLE
question about board compatibility

### ISSUE THREAD

**Issue by ntzb** [Pako2/esp-oledproc#1]
hi! thanks for your awesome lib.
my question is about board compatibility.
you wrote that 
> ESP32 does not supported for now

is this because of the flash size requirement (4MB)? or esp8266 vs esp32 differences?
I have esp32-wroom-32d (flash size 4MB), need to know if it can work, or else I need to buy another part.

thanks!

**Comment by Pako2** [comment]
Hi !
The above sentence about ESP32 means only one thing. At the time when I was working on the project, I had no ESP32 board at my disposal, so of course I could not perform the necessary tests. I'm sure it can work with ESP32 too, but it will definitely need some minor modifications.
In the meantime, I used ESP32 several times in other projects and even used the UTF-8 text scrolling function there (which I created in the "oledproc" project). While adapting scrolling for another project, I discovered that the function contains some bug (for which I already know the fix) and I plan to go back to "oledproc" and fix it. In doing so, I might as well add ESP32 support. However, I cannot promise when it will be.

**Comment by ntzb** [comment]
are the modifications related to the IO pins? or is it something else?
can you point me to it?

**Comment by Pako2** [comment]
Yes, I meant GPIO in particular. But there will be more. E.g. the platformio.ini file must definitely be edited.
And there will be other things - coincidentally just yesterday I came across a small difference between ESP8266 and ESP32 in OTA via the web (ESP32 does not use "Update.runAsync(true);" command).
At this moment I am definitely not able to list everything. You just need to start and the compiler will point out the differences.
